test/Interop/Cxx/class/loadable-types-silgen.swift

```diff
@@ -0,0 +1,93 @@
+// RUN: %target-swift-emit-silgen -I %S/Inputs -enable-cxx-interop %s | %FileCheck %s
+
+import LoadableTypes
+
+// CHECK-LABEL: sil hidden [ossa] @$s4main21passAddressOnlyStruct{{[_0-9a-zA-Z]*}}F
+func passAddressOnlyStruct(s: EmptyStruct) {
```

Comment:
The function name does not match the intent :)

Reply:
Done